### PR TITLE
Dashboard visual enchancement

### LINKED ISSUE #1257: Dashboard product list visual improvements
- [x] Set padding for "X item(s) selected" text
- [x] Add translation for text "X item(s) selected"
- [x] Hide bulk action select if no items are selected
- [x] Remove circular border around product image thumbnails
- [x] Make padding around thumbnails smaller

### PR DESCRIPTION
**Pull request by dominik-zeglen**
Resolves #1257 and #1258

### CODE REVIEW

**Comment by patrys** [comment]
Please attach screenshots when UI changes 🙏

**Comment by dominik-zeglen** [comment]
This is how UI looks after fixes:
zrzut ekranu 2017-11-07 o 09 57 48

File: saleor/static/dashboard/js/components/bulk-actions.js [hunk 1]
```diff
@@ -1,8 +1,18 @@
-$(document).ready((e) => {
-  function updateCounterSelected() {
-    const count = $('.select-item:checked').length;
-    $('.data-table-header-action-selected-items #counter').text(count);
+function updateCounterSelected () {
+  const count = $('.select-item:checked').length;
+  const $counterTextNode = $('.data-table-header-action-selected-items');
+  let counterText;
+
+  if (count === 1) {
```
Comment (elwoodxblues) [inline saleor/static/dashboard/js/components/bulk-actions.js]: Django provides API for this: [ngettext](https://docs.djangoproject.com/en/1.11/topics/i18n/translation/#ngettext)

**Comment by patrys** [comment]
Could we maybe:

* inline the number of items selected with the action form
* inline the various possible bulk actions as buttons that are hide when nothing is selected

**Comment by patrys** [comment]
Prior art (scroll to "Alternative headers"): https://material.io/guidelines/components/data-tables.html#data-tables-tables-within-cards

**Comment by elwoodxblues** [comment]
I was reluctant to introduce bulk actions as inline buttons since that works only if we have a few actions. The select widget is a bit more universal solution. But maybe that's something that people would customize when they need more actions.

**Comment by patrys** [comment]
If we have more actions than fit on screen then we're facing a much bigger problem 😉

**Comment by elwoodxblues** [comment]
Perhaps, but I was also thinking about facing client requirements 😄 But I agree, that's a bigger UX problem then.

**Comment by elwoodxblues** [comment]
@dominik-zeglen We have to add `ngettext` to `.eslintrc.json` file.

**Pull request merged**